doc/nrf/protocols/matter/index.rst

```diff
@@ -23,9 +23,9 @@ For a full list of |NCS| and Matter versions, view the following table:
    +--------------------------+-----------------------------------------------------+------------------------+
    | nRF Connect SDK version  | Matter specification version                        | Matter SDK version     |
    +==========================+=====================================================+========================+
-   | v2.8.99 (latest)         | :ref:`1.3.0 <ug_matter_overview_dev_model_support>` | 1.3.0.0                |
-   +--------------------------+                                                     |                        |
-   | |release|                |                                                     |                        |
+   | v2.8.99 (latest)         | :ref:`1.4.0 <ug_matter_overview_dev_model_support>` | 1.4.0.0                |
```

Comment:
Need to also edit Line 18.

Reply:
Are you sure? I haven't done it, because it mentions NCS release, which is 2.8.0 and this one is based on 1.3.0.0, so I thought we are going to change it near the next 2.9.0 release.

Reply:
Ah, right you are! I was just looking at the matter version, but you are right, it specifies the latest release.